### PR TITLE
[DOCS] Add link to Elasticsearch labs ELSER Python notebook

### PR DESCRIPTION
**Pull request by leemthompo**
Link to Elasticsearch Labs ELSER notebook!

### CODE REVIEW

File: docs/reference/search/search-your-data/semantic-search-elser.asciidoc [hunk 1]
```diff
@@ -361,3 +361,9 @@ PUT my-index
 * {ml-docs}/ml-nlp-elser.html[How to download and deploy ELSER]
 * {ml-docs}/ml-nlp-limitations.html#ml-nlp-elser-v1-limit-512[ELSER v1 limitation]
 * https://www.elastic.co/blog/may-2023-launch-information-retrieval-elasticsearch-ai-model[Improving information retrieval in the Elastic Stack: Introducing Elastic Learned Sparse Encoder, our new retrieval model]
+
+[discrete]
+[[further-reading]]
+==== Interative example
```
Comment (leemthompo) [inline docs/reference/search/search-your-data/semantic-search-elser.asciidoc:367]:
```suggestion
[[interactive-example]]
==== Interactive example
```

File: docs/reference/search/search-your-data/semantic-search-elser.asciidoc [hunk 2]
```diff
@@ -361,3 +361,9 @@ PUT my-index
 * {ml-docs}/ml-nlp-elser.html[How to download and deploy ELSER]
 * {ml-docs}/ml-nlp-limitations.html#ml-nlp-elser-v1-limit-512[ELSER v1 limitation]
 * https://www.elastic.co/blog/may-2023-launch-information-retrieval-elasticsearch-ai-model[Improving information retrieval in the Elastic Stack: Introducing Elastic Learned Sparse Encoder, our new retrieval model]
+
+[discrete]
+[[interactive-example]]
+==== Interactive example
+
+* The `elasticsearch-labs` repo has an interactive example of running https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/search/03-ELSER.ipynb[ELSER-powered semantic search] using the Elasticsearch Python client.
```
Comment (szabosteve) [inline docs/reference/search/search-your-data/semantic-search-elser.asciidoc:369]:
```suggestion
* The `elasticsearch-labs` repo has an interactive example of running https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/search/03-ELSER.ipynb[ELSER-powered semantic search] using the {es} Python client.
```

**Review by szabosteve — APPROVED** [review]
LGTM, thank you! I left a tiny, non-blocking suggestion. Please take or leave it.

File: docs/reference/search/search-your-data/semantic-search-elser.asciidoc (hunk 2 above)
Comment (leemthompo) [reply docs/reference/search/search-your-data/semantic-search-elser.asciidoc:369]: Thanks! Need to get in habit of using {es} variable!

**Comment by leemthompo** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.10|[](https://github.com/elastic/elasticsearch/pull/99080)|
|✅|8.9|[](https://github.com/elastic/elasticsearch/pull/99081)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**